src/backend/expungeservice/test/test_expunger.py

```diff
@@ -20,16 +20,51 @@ def test_statute():
 
 def get_disp():
     date = datetime.date(1996, 1, 1)
-    disp_type = DispositionType.NO_CONVICTION
+    disp_type = DispositionType.CONVICTED
```

Comment:
One thing to do would be to name the functions better. get_disp() doesn't clearly state what it's doing which is creating a Ten plus year convicted disposition. If arguments are passed to the function to create the dispo than it makes sense to have a broader name.

Reply:
Good point. Will update.